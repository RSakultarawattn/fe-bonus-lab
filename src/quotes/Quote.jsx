import React from 'react';
import PropTypes from 'prop-types';

const Quote = ({ content }) => (
  <figure>
    
    <p>{content}</p>

  </figure>
);

Quote.propTypes = {
  content: PropTypes.string.isRequired
};

export default Quote;
