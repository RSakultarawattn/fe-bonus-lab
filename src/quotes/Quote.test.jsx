import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Quote from './Quote';

describe('displays a quote', () => {
  afterEach(() => cleanup());
  it('renders one single quote', () => {
    const { asFragment } = render(<Quote 
      content="" faction="" id="" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
