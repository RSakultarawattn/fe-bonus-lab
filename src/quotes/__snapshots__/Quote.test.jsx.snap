// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays a quote renders one single quote 1`] = `
<DocumentFragment>
  <figure>
    <p />
  </figure>
</DocumentFragment>
`;
